docs/validators/staking/unjailing.md

```diff
@@ -101,59 +101,59 @@ You can write the text for the "Data" field for _any_ number of nodes. The gener
 unJail@<BLS1>@<BLS2>@…@<BLS99>
 ```
 
-## **Unjailing through erdpy**
+## **Unjailing through mxpy**
 
-Submitting the unjailing transaction using `erdpy` avoids having to write the "Data" field manually. Instead, the transaction is constructed automatically by `erdpy` and submitted to the network directly, in a single command.
+Submitting the unjailing transaction using `mxpy` avoids having to write the "Data" field manually. Instead, the transaction is constructed automatically by `mxpy` and submitted to the network directly, in a single command.
 
-Make sure `erdpy` is installed by issuing this command on a terminal:
+Make sure `mxpy` is installed by issuing this command on a terminal:
 
 ```python
-erdpy --version
+mxpy --version
 ```
 
-The version reported by this command must be at least `erdpy 0.7.0`, or higher. If `erdpy` is not installed (`command not found`), or if the version is lower than `0.7.0`, please follow [these instructions](/sdk-and-tools/erdpy/installing-erdpy).
+The version reported by this command must be at least `mxpy 0.7.0`, or higher. If `mxpy` is not installed (`command not found`), or if the version is lower than `0.7.0`, please follow [these instructions](/sdk-and-tools/mxpy/installing-mxpy).
```

Comment:
maybe increase the mxpy version here as well

Reply:
Fixed.